acceptance/setup/packages/pre-suite/010_Install.rb

```diff
@@ -48,5 +48,15 @@
 install_packages_on(master, MASTER_PACKAGES)
 install_packages_on(agents, AGENT_PACKAGES)
 
+agents.each do |agent|
+  if agent['platform'] =~ /windows/
+    arch = agent[:ruby_arch] || 'x86'
+    base_url = "http://builds.puppetlabs.lan/puppet/#{ENV['SHA']}/artifacts/windows"
+    filename = "puppet-agent-#{ENV['VERSION']}-#{arch}.msi"
```

Comment:
Make sure that this follows the convention of puppet-agent, which should be something like:

`puppet-agent-0.2.4.8.g7cdf605-x86.msi`